rosidl_typesupport_connext_c/package.xml

```diff
@@ -10,14 +10,14 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>connext_cmake_module</buildtool_depend>
   <buildtool_depend>rosidl_cmake</buildtool_depend>
-  <buildtool_depend>rosidl_generator_c</buildtool_depend>
+  <buildtool_depend>rosidl_runtime_c</buildtool_depend>
   <buildtool_depend>rosidl_typesupport_connext_cpp</buildtool_depend>
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>connext_cmake_module</buildtool_export_depend>
   <buildtool_export_depend>rcutils</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
-  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
```

Comment:
There is still code that references `rosidl_generator_c`:

https://github.com/ros2/rosidl_typesupport_connext/blob/2a408f8f410c68c9b67ece28d245fcbaefbfc3d5/rosidl_typesupport_connext_c/resource/msg__type_support_c.cpp.em#L4-L6

Shouldn't we continue to depend on it in the manifest?

Reply:
See https://github.com/ros2/rosidl_typesupport_connext/pull/54 for a proposal to add back the dependency.